src/components/DivMotion.tsx

```diff
@@ -0,0 +1,15 @@
+import { motion } from "framer-motion";
+import type { ReactNode } from "react";
+
+export const DivMotion = ({ children }: { children: ReactNode }) => {
```

Comment:
```suggestion
export const MotionDiv = ({ children }: { children: ReactNode }) => {
```

Thoughts on renaming to `MotionDiv` for consistency?